extension/app/ts/components/subcomponents/CopyToClipboard.tsx

```diff
@@ -21,3 +21,21 @@ export function CopyToClipboard(props: Props): preact.VNode {
 		</div>
 	)
 }
+
+interface ToolTipParams {
+	children: preact.VNode
```

Comment:
Should this be an array?  Also, I think you want `ComponentChildren` here?
https://preactjs.com/guide/v10/typescript#function-components

Reply:
yeah, seems like that works too. I replaced it in other places too